README.md

```diff
@@ -0,0 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# portfolio project
+
+> this is my first portifolio.
+
+![screenshot](./images/screenshot/screenshot-mobile-version.png)
+
+home page for mobile skeleton
+
+## Built With
+
+- HTML 
+- CSS
+- No framwork used in this project
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] You can remove the 'No framwork used in this project' text as there is no need to mention it here.

- [ ] Your demo link is not working properly. You can set up the live link like this:
- Go to this link (https://github.com/alexisuwimana/Portfolio-mobile-version-skeleton/settings/pages).
- Select the branch name
- Select `root`
- Click the `Save` button

Your project will be deployed by Github Pages